### PR TITLE
Fixed Combobox Group Label Token bug

### PR DESCRIPTION
**Pull request by vinitkumargoel**
## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/components/combobox/scss/combobox.scss [hunk 1]
```diff
@@ -172,13 +172,13 @@ ul {
     font-weight: 400;
     font-size: 14px;
     line-height: 19px;
-    color: var(--combobox-group-label-color);
+    color: var( --combobox-groupLabel-color);
     padding: 9px 16px;
-    border-top: 1px solid var(--combobox-group-label-border-color);
-    border-bottom: 1px solid var(--combobox-group-label-border-color);
+    border-top: 1px solid var(--combobox-groupLabel-border-color);
```
Comment (merajcal) [inline web-components/src/components/combobox/scss/combobox.scss:177]: why did we change to camelcase (groupLabel)? ,

**Review by merajcal — CHANGES_REQUESTED** [review]

File: web-components/src/components/combobox/scss/combobox.scss (hunk 1 above)
Comment (vinitkumargoel) [reply web-components/src/components/combobox/scss/combobox.scss]: @merajcal Fixed

**Review by merajcal — APPROVED** [review]

File: web-components/src/components/combobox/scss/combobox.scss [hunk 2]
```diff
@@ -172,7 +172,7 @@ ul {
     font-weight: 400;
     font-size: 14px;
     line-height: 19px;
-    color: var(--combobox-group-label-color);
+    color: var( --combobox-group-label-color);
```
Comment (parsoman) [inline web-components/src/components/combobox/scss/combobox.scss:175]: The extra space is not required

**Comment by vinitkumargoel** [comment]
BUILD

File: web-components/src/components/combobox/scss/combobox.scss (hunk 2 above)
Comment (vinitkumargoel) [reply web-components/src/components/combobox/scss/combobox.scss]: FIXED

**Review by parsoman — APPROVED** [review]

**Review by merajcal — DISMISSED** [review]

**Comment by vinitkumargoel** [comment]
BUILD

**Comment by vinitkumargoel** [comment]
BUILD

**Comment by vinitkumargoel** [comment]
Build

**Comment by vinitkumargoel** [comment]
BUILD

**Comment by vinitkumargoel** [comment]
BUILD

**Comment by vinitkumargoel** [comment]
BUILD

**Review by parsoman — APPROVED** [review]

**Pull request merged**